eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -198,6 +207,21 @@ noSuchGovActions gas gaIds =
    in failureUnless (Set.null unknownGovActionIds) $
         GovActionsDoNotExist unknownGovActionIds
 
+checkVotesAreValid ::
+  ConwayEraPParams era =>
+  Map.Map (GovActionId (EraCrypto era)) (GovActionState era) ->
+  VotingProcedures era ->
+  Test (ConwayGovPredFailure era)
+checkVotesAreValid gas (VotingProcedures vps) =
+  let go voter gmap =
+        let foldMapFailures = flip foldMap' . Map.toList . fmap gasAction . Map.restrictKeys gas $ Map.keysSet gmap
```

Comment:
I found this line very difficult to understand, though maybe that speaks more about me than about the code! 

In any case, the line is too long, so maybe you can split it in two lines, or if you wanted to split it semantically, maybe like this:
```
        let
            votedActions = fmap gasAction (Map.restrictKeys gas (Map.keysSet gmap))
            foldMapFailures = flip foldMap' ( Map.toList votedActions)
```

Reply:
Thank you!